dev/benchmarks/microbenchmarks/lib/foundation/decode_and_parse_asset_manifest.dart

```diff
@@ -2,32 +2,30 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:convert';
+import 'dart:typed_data';
 
-import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart' show PlatformAssetBundle;
 import 'package:flutter/widgets.dart';
 
 import '../common.dart';
 
-const int _kNumIterations = 1000;
+const int _kNumIterations = 1;
```

Comment:
Maybe 10-ish iterations?

Reply:
Oops, this was in error. I should keep it at 1000 to maintain parity with the current version of the benchmark, no?

Reply:
Yes, good idea